settings/base.py

```diff
@@ -3,7 +3,6 @@
 """
 
 import os
-from celery import Celery
 
 DEBUG = True
 TEMPLATE_DEBUG = DEBUG
```

Comment:
Also, STATICFILES_DIRS and STATICFILES_FINDERS can be removed.

https://docs.djangoproject.com/en/1.11/ref/settings/#std:setting-STATICFILES_FINDERS